Userland/Utilities/acpi.cpp

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2022-2022, Andreas Kling <kling@serenityos.org>
```

Comment:
You can put your name here or if you don't want:
`Copyright (c) 2022, the SerenityOS developers.`